NIRoberto/myband
# Document my API using swagger

1. What does this PR do?

- Document My API using Swagger 

2. Description of Task to be completed

- Document my API using swagger

3. How should this be manually tested?

-  Use this link 

-  Scroll down above the readme.md text you will see those reports  

4. Any background context you want to provide?

5.What is the relevant Trello story?

-  https://trello.com/c/g1lBFsIt

6.Screenshots (if appropriate)
